bin/build.js

```diff
@@ -12,3 +12,15 @@ esbuild.build({
     mainFields: ['module', 'main'],
     watch: shouldWatch,
 }).catch(() => process.exit(1))
+
+esbuild.build({
```

Comment:
Not sure about this tho. It works for me, but idk if thats the way to go.

I'll leave the build process up to you 😛